spec/spec.md

```diff
@@ -167,27 +167,28 @@ A [[ref:money address]] is a payment-network-specific identifier embedded within
 sending or receiving funds in a particular currency. It is represented by the following URN [[spec:RFC8141]] structure:
 
 ```
-urn:<currency>:<curr_specific_part>  
+urn:<currency>:<curr_specific_part>:<ttl>
```

Comment:
seems weird to apply a ttl in a urn itself, given that it can be manipulated

can it instead be put in some metadata around the document when it's resolved?

Reply:
yeah the reason for this was did doc service types wouldn't recognize it especially strongly typed did libs. but I'm not sold on this approach yet either so  ima close this pr and gonna continue discussion in #11 if you had more thoughts!